### PR TITLE
harden RemoteRestartedQuarantinedSpec, #17314

### PR DESCRIPTION
**Pull request by patriknw**
* Seems like the initial message may be dropped in this case,
  we are at-most once after all. See detailed logs in issue.

Refs #17314

### CODE REVIEW

**Comment by patriknw** [comment]
since this is failing frequently in akka-artery-perf job I would like to try this rather quickly

**Review by johanandren — APPROVED** [review]
LGTM

**Comment by patriknw** [comment]
thanks

File: akka-remote-tests/src/multi-jvm/scala/akka/remote/RemoteRestartedQuarantinedSpec.scala [hunk 1]
```diff
@@ -130,13 +130,13 @@ abstract class RemoteRestartedQuarantinedSpec
...
+        // retry because it's possible to loose the initial message here, see issue #17314
         val probe = TestProbe()(freshSystem)
-
-        freshSystem.actorSelection(RootActorPath(firstAddress) / "user" / "subject").tell(Identify("subject"), probe.ref)
-        // TODO sometimes it takes long time until the new connection is established,
-        //      It seems like there must first be a transport failure detector timeout, that triggers
-        //      "No response from remote. Handshake timed out or transport failure detector triggered".
-        probe.expectMsgType[ActorIdentity](30.second).ref should not be (None)
+        probe.awaitAssert({
+          println(s"# --") // FIXME
```
Comment (patriknw) [inline akka-remote-tests/src/multi-jvm/scala/akka/remote/RemoteRestartedQuarantinedSpec.scala]: ops, this shouldn't have been included

**Pull request merged**